src/control/server/mgmt_drpc.go

```diff
@@ -84,3 +86,23 @@ func (mod *srvModule) handleNotifyReady(reqb []byte) error {
 
 	return nil
 }
+
+func (mod *srvModule) handleBioErr(reqb []byte) error {
+	req := &srvpb.BioErrorReq{}
+	if err := proto.Unmarshal(reqb, req); err != nil {
+		return errors.Wrap(err, "unmarshal BioError request")
+	}
+
+	if req.InstanceIdx >= uint32(len(mod.iosrvs)) {
+		return errors.Errorf("instance index %v is out of range (%v instances)",
+			req.InstanceIdx, len(mod.iosrvs))
+	}
+
+	if err := checkDrpcClientSocketPath(req.DrpcListenerSock); err != nil {
+		return errors.Wrap(err, "check BioErr request socket path")
+	}
+
+	mod.iosrvs[req.InstanceIdx].BioErrorNotify(req)
```

Comment:
at some point I presume we are going to want to do something more than just log debug message for these events, something like adding to an event store that lives on the management service?

Reply:
I haven't thought very much about this, maybe we can discuss this option further?